docs/docs/scanner/misconfiguration/custom/index.md

```diff
@@ -120,21 +119,50 @@ Trivy supports extra fields in the `custom` section as described below.
+| scope                      | `package`                                                         | Defines the policy scope                                |  (applied to all input types) | :material-close: | :material-check: |     :material-check:            |
+| custom.id                  | Any characters                                                    | More information provided below                         |              N/A              | :material-close: | :material-check: |     :material-check:            |
+| custom.avd_id              | Any characters                                                    | More information provided below                         |              N/A              | :material-close: | :material-check: |     :material-check:            |
+| custom.severity            | `LOW`, `MEDIUM`, `HIGH`, `CRITICAL`                               | The severity of the misconfiguration                    |            `MEDIUM`           | :material-close: | :material-check: |     :material-check:            |
+| custom.provider            | Any provider available in Trivy                                   | The resource provider this check relates to             |              N/A              | :material-close: | :material-check: |     :material-close:            |
+| custom.service             | The service name of the provider                                  | One of the services available in the provider           |              N/A              | :material-close: | :material-check: |     :material-close:            |
+| custom.short_code          | Any characters                                                    | Descriptive name for the check                          |              N/A              | :material-close: | :material-check: |     :material-check:            |
+| custom.subtypes            | Cloud Resource Subtypes                                           | Refer to the section on subtypes below                  |              N/A              | :material-close: | :material-check: |     :material-close:            |
+| custom.recommended_actions | Any characters                                                    | Describing what the user should do to resolve the issue |              N/A              | :material-close: | :material-check: |     :material-check:            |
+| custom.input.selector.type | Any item(s) in [this list][source-types]                          | More information provided below                         |              N/A              | :material-close: | :material-check: |     :material-check:            |
+| related_resource           | Any characters                                                    | URL to related resources                                |              N/A              | :material-close: | :material-check: |     :material-check:            |
+
```

Comment:
@simar7  could you please check this table, specifically the "In Table" and "In JSON" parts?

Reply:
TBH I don't understand what's the "In table" and "JSON" are for. Could you clarify?

The table is quite wide and not very easy to view as you can see below. Should we split into two table assuming we need all the columns? But nevertheless, we should move the "Required" column to the left as it's probably the most important for users to know.

<img width="920" alt="image" src="https://github.com/aquasecurity/trivy/assets/1254783/6c6f4fb5-223c-4864-9002-983c4c93c704">

Reply:
@knqyf263 could you please clarify what "in table" and "in json" is as we are not sure what that means? I assume the output format but not sure how it relates to writing policies?

Reply:
Oki, thank you

Reply:
@AnaisUrlichs I saw the review request but I still see the JSON/table columns in the PR. Did you forget to push the change?